snap/snapcraft.yaml

```
@@ -390,16 +390,52 @@ parts:
     plugin: nil
     override-prime: |
       mkdir -p ${CRAFT_PRIME}/snap
+
+      # Avoid new userspace drivers slipping into the consumer snaps,
+      # using wildcards to support multi-arch snaps
+      cat <<EOF > ${CRAFT_PRIME}/snap/${CRAFT_ARCH_BUILD_FOR}.list
+      usr/lib/*/dri/*
+      usr/lib/*/libVkLayer_*.so
+      usr/lib/*/libvulkan_*.so
+      usr/lib/*/vdpau/*
+      usr/share/drirc.d/*
+      usr/share/egl/egl_external_platform.d/*
+      usr/share/glvnd/egl_vendor.d/*
+      usr/share/libdrm/*
+      usr/share/vulkan/*.d/*
+      EOF
+
+      # Elements that should be pruned from the list
+      LIST_CLEANUP=(
```

We're not intending to export this?

```suggestion
      cleanup_patterns=(
```

---

Good one. Done.